docs/glossary.mdx

```diff
@@ -0,0 +1,130 @@
+You can refer to our [how-to guide](./how_to_guides/manifest_annotations.mdx) to understand how ORAS CLI could be used to add them.
+
+### Artifacts
+
+Artifacts are a conceptual piece of content stored as [Blobs](#blob) with an accompanying Manifest containing a [Config](#config). 
+We can [push](./commands/oras_push.mdx), 
+[pull](./commands/oras_pull.mdx), 
+[attach](./commands/oras_attach.mdx) artifacts using the ORAS CLI. 
+In order to understand the usage better, 
+you may follow the steps in our [quick start guide](./quickstart.mdx).
+
+### Blob
+
+Blob (which stands for Binary Large Objects) is the content stored by a registry and is addressable by a Digest. 
+ORAS allows you to [fetch](./commands/oras_blob_fetch.mdx), 
+[delete](./commands/oras_blob_delete.mdx) 
+and [push](./commands/oras_blob_push.mdx) blobs.
+
+### Config
+
+[Config](https://github.com/opencontainers/image-spec/blob/main/config.md) is the JSON format describing images for use with a container runtime and execution tool and its relationship to filesystem changesets. 
+You may use [`oras manifest fetch-config`](./commands/oras_manifest_fetch-config.mdx) to check out the config of your artifact.
+
+### Container Images
+
+A [container image](https://www.docker.com/resources/what-container/#:~:text=A%20Docker%20container%20image%20is,tools%2C%20system%20libraries%20and%20settings.) is a small, standalone, 
+executable file that contains all the components required to run an application, 
+including the code, runtime, system tools, system libraries, and settings.
+
+### Content Descriptors
+
+A [Content Descriptor](https://github.com/opencontainers/image-spec/blob/main/descriptor.md) (or simply Descriptor) describes the disposition of the targeted content. 
+It includes the type of the content, 
+a content identifier (digest), 
+and the byte-size of the raw content. 
+Optionally, it includes the type of artifact it is describing.
+
+### Digest
+
+A [digest](https://github.com/opencontainers/image-spec/blob/main/descriptor.md#digests) is a property of a Descriptor which uniquely identifies content by taking a collision-resistant hash of the bytes.
+
+### Distribution-spec
+
+The [OCI Distribution Spec](https://github.com/opencontainers/distribution-spec/blob/main/spec.md) defines an API protocol to facilitate and standardize the distribution of content.
+
+### Extensions
+
+The Distribution Specification Project includes a process and API for prototyping and testing [extensions](https://github.com/opencontainers/distribution-spec/blob/main/extensions/README.md) to the Distribution API.
+
+### Image Layout
+
+The [OCI Image Layout](https://github.com/opencontainers/image-spec/blob/main/image-layout.md) is the directory structure for OCI content-addressable blobs and location-addressable references (refs). 
+It must contain a `blobs` directory, `oci-layout` file and an `index.json` file. 
+To learn more, check out the how-to guide on OCI Layouts.
```

Comment:
Should add this once the other PR is complete?

Reply:
@sajayantony yes, we should wait for that PR to merge.

Reply:
Which PR are we waiting for here?

Reply:
@TerryHowe didn't notice the PR got merged, I'll update the link in the glossary and it should be ready to merge.